<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>685</width>
    <height>413</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>16</x>
      <y>19</y>
      <width>651</width>
      <height>331</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tabUsers">
     <attribute name="title">
      <string>Users</string>
     </attribute>
     <widget class="QWidget" name="horizontalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>9</x>
        <y>9</y>
        <width>631</width>
        <height>281</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="layoutUsers">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,0,0">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <widget class="QComboBox" name="userGroupSelector"/>
         </item>
         <item>
          <widget class="QPushButton" name="addGroup">
           <property name="text">
            <string>Add group</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="removeGroup">
           <property name="text">
            <string>Remove grp</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QListWidget" name="listUsers"/>
       </item>
       <item>
        <layout class="QVBoxLayout" name="layoutUsersActions">
         <item>
          <widget class="QLabel" name="username">
           <property name="text">
            <string>Username</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QListWidget" name="listUserGroups"/>
         </item>
         <item>
          <widget class="QListWidget" name="listUserPermissions"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tabGroups">
     <attribute name="title">
      <string>Groups</string>
     </attribute>
     <widget class="QWidget" name="horizontalLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>631</width>
        <height>281</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="layoutGroups">
       <item>
        <widget class="QListWidget" name="listGroups"/>
       </item>
       <item>
        <layout class="QVBoxLayout" name="layoutGroupsActions">
         <item>
          <widget class="QLabel" name="groupname">
           <property name="text">
            <string>GroupName</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QListWidget" name="listGroupPermissions"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="errorMessage">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>651</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>685</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Fi&amp;le</string>
    </property>
    <addaction name="actionConnect"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionRead_the_RFC"/>
    <addaction name="actionAccess_the_wiki"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionRead_the_RFC">
   <property name="text">
    <string>Read the RFC</string>
   </property>
  </action>
  <action name="actionAccess_the_wiki">
   <property name="text">
    <string>Access the wiki</string>
   </property>
  </action>
  <action name="actionConnect">
   <property name="text">
    <string>Connect</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
